service-commands/scripts/allClockValues.js

```diff
@@ -0,0 +1,152 @@
+const fs = require('fs')
+
+const { map } = require('lodash')
+const axios = require('axios')
+const retry = require('async-retry')
+async function getUsers(discoveryProvider, offset, limit) {
+  return (
+    await retry(
+      () =>
+        axios({
+          method: 'get',
+          url: '/users',
+          baseURL: discoveryProvider,
+          params: { offset, limit, is_creator: true }
```

Comment:
Yes, I noticed this as well, we can get past this with discovery provider changes that sort before sending the data.

We can run a hotfix branch to do this as well rather than rolling it out for all nodes.